docs/interactions/structured-messages/webview-sdk.md

```diff
@@ -88,9 +88,10 @@ Since not every channel supports every feature, you'll find support info in the
 * **<span style="color:orange">Not yet</span>**: Supported by channel but not by SDK yet
 * N/A: Not supported by channel
 
-|       | **Engage Messaging Web** | **Engage Messaging iOS** | **Engage Messaging Android** | **Unsupported channels** |
+| | **Engage Messaging Web** | **Engage Messaging iOS** | **Engage Messaging Android** | **Unsupported channels** |
 |-|-|-|-|-|
 | **close** | <span style="color: green">Yes</span> | <span style="color: orange">Not yet</span> | <span style="color: orange">Not yet</span> | N/A |
+| **Automatic title update** | <span style="color: green">Yes</span> | <span style="color: orange">Not yet</span> | <span style="color: orange">Not yet</span> | N/A |
```

Comment:
Is it possible to link these features to the section where they are described? So this one to the section just below, and the close one to the section just above.

Reply:
Yes, I'll create anchors